### PR TITLE
feature: update `delete-dev.sh` to remove DB volumes by default

### LINKED ISSUE #1802: delete-dev.sh does not delete database
### What Operating System(s) are you seeing this problem on?

Linux (x86-64)

### Backend.AI version

main

### Describe the bug

https://github.com/lablup/backend.ai/blob/8ef6fdd0dae39f499dc3e3f7c363be103e0f175a/scripts/install-dev.sh#L221C59-L221C59
According to the output of install-dev.sh, delete-dev.sh deletes the database.
However, even if delete-dev.sh is actually executed, all contents of volumes remain.

### To Reproduce

_No response_

### Expected Behavior

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by minseokey**
This PR resolves #1802 by adding option "--skip-db".

1. User can delete all data volumes with no option.
`./scripts/delete-dev.sh`
2. User can skip to delete all data volumes with "--skip-db" option.
`./scripts/install-dev.sh --skip-db`
3. User can get help through the --help command.
`./scripts/install-dev.sh --help`
and then,
  ```
  USAGE
    ./scripts/delete-dev.sh [OPTIONS]
  
  OPTIONS
    -h, --help         Show this help and exit
  
    --skip-containers  Skip removal of docker resources (default: false)
  
    --skip-venvs       Skip removal of temporary virtualenvs (default: false)
  
    --skip-db       Skip removal of volume resources (default: false)
  ```
**Checklist:** (if applicable)

- [ ] Milestone metadata specifying the target backport version
- [ ] Mention to the original issue
- [ ] Installer updates including:
  - Fixtures for db schema changes
  - New mandatory config options
- [ ] Update of end-to-end CLI integration tests in `ai.backend.test`
- [ ] API server-client counterparts (e.g., manager API -> client SDK)
- [ ] Test case(s) to:
  - Demonstrate the difference of before/after
  - Demonstrate the flow of abstract/conceptual models with a concrete implementation
- [ ] Documentation
  - Contents in the `docs` directory
  - docstrings in public interfaces and type annotations

### CODE REVIEW

File: scripts/delete-dev.sh [hunk 1]
```diff
@@ -34,6 +34,8 @@ usage() {
   echo "  ${LWHITE}--skip-containers${NC}  Skip removal of docker resources (default: false)"
   echo ""
   echo "  ${LWHITE}--skip-venvs${NC}       Skip removal of temporary virtualenvs (default: false)"
+  echo ""
+  echo "  ${LWHITE}--skip-db${NC}       Skip removal of volume resources (default: false)"
```
Comment (rapsealk) [inline scripts/delete-dev.sh]: Please maintain consistent indentation.

File: scripts/delete-dev.sh [hunk 2]
```diff
@@ -134,6 +138,11 @@ else
   show_info "Skipped removal of Docker containers."
 fi
 
+if [ $REMOVE_DB -eq 1 ]; then
+  show_info "Removing data volumes..."
+  sudo rm -rf volumes
```
Comment (rapsealk) [inline scripts/delete-dev.sh]: This command does not need to be run in `sudo` mode.

**Comment by rapsealk** [comment]
Thanks for the work!
Please set `Projects` and `Milestone` on the right panel.
| Projects | Milestone |
| -------- | ---------- |
| image | image |

**Comment by rapsealk** [comment]
I recommend updating this line, as the database will be deleted only if the option is enabled.

https://github.com/lablup/backend.ai/blob/44612158c9a4e23a0d215a5a859cf52c7a43cfc5/scripts/install-dev.sh#L220-L221

**Comment by minseokey** [comment]
@rapsealk My intention is to delete it if there is no option, and skip the deletion when you put the option in.

**Comment by rapsealk** [comment]
@minseokey Ouch, my apologies for the misunderstanding.

**Pull request merged**